packages/round-manager/src/features/round/ViewApplicationPage.tsx

```diff
@@ -221,11 +282,11 @@ export default function ViewApplicationPage() {
               <div className="grid sm:grid-cols-3 gap-2 md:gap-10">
                 <div className="text-grey-500 truncate block">
                   <MailIcon className="inline-flex h-4 w-4 text-grey-500 mr-1"/>
-                  <span className="text-xs text-grey-400">{ getAnswer("Email", application) }</span>
+                  <span className="text-xs text-grey-400">{ getAnswer("Email") }</span>
```

Comment:
can all the question be defined in `ApplicationQuestions` enum so here could be `ApplicationQuestions.EMAIL`?